setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='scylla-api',
-    version='0.0.1',    # Will be moved later to be set as part of the CI pipeline
+    version='1.0',  # Will be moved later to be set as part of the CI pipeline
```

Comment:
Is the comment still relevant?

Reply:
Yes, we should move `version` to be dynamic based on latest git tag

Reply:
@benipeled I would recommend https://pypi.org/project/setuptools-scm/

Reply:
also keep in mind once you uploaded a version to pypi, you can't upload same version number again. (and you can't delete it from pypi)

Reply:
Thanks, that's one of the options I'm checking.